pkg/adapter/apiserver/delegate.go

```diff
@@ -44,6 +45,12 @@ func (a *resourceDelegate) Add(obj interface{}) error {
 
 	if result := a.ce.Send(context.Background(), event); !cloudevents.IsACK(result) {
 		a.logger.Errorw("failed to send event", zap.Error(result))
+	} else {
+		eventExt, err := json.Marshal(event.Extensions())
+		if err != nil {
+			a.logger.Error(err)
+		}
+		a.logger.Infof("event sent for resource %s", string(eventExt))
```

Comment:
wait wait wait, the identifying info for the resource is in the source+subject. Why are you printing the extensions?